Documentation/docs/learn/faq.md

```diff
@@ -346,7 +346,7 @@ be summarized as follows:
  guidelines for the development of software-related medical devices."
 ```
 
-For mode details see the page [FDA Guidelines for Software Developement](https://itk.org/Wiki/FDA_Guidelines_for_Software_Development).
+For mode details see the page [FDA Guidelines for Software Developement](fda_off_the_shelf_sw_guidelines.md).
```

Comment:
Thanks Jon. While the intention is fine to me, I still find this link a bit confusing. It seems to me that the .md file should have been named something like `fda_guidelines_for_software_development.md` instead, without the term "off the shelf". Looking at the original, https://itk.org/Wiki/FDA_Guidelines_for_Software_Development, the title is "FDA Guidelines for Software Development". (It has an "off the shelf" section, but it has other sections as well.)

Reply:
OK, but this patch set removes redirections to the wiki for the integrated documents; I will rename the file and change the hyperlink in a separate PR.

Reply:
@N-Dekker PR #4374.